core/src/test/java/org/apache/iceberg/rest/TestRESTCatalog.java

```diff
@@ -1742,4 +1753,129 @@ public void testCatalogTokenRefreshDisabledWithCredential() {
             eq(catalogHeaders),
             any());
   }
+
+  @Test
+  public void multipleDiffsAgainstSingleTable() {
+    Namespace namespace = Namespace.of("namespace");
+    TableIdentifier identifier = TableIdentifier.of(namespace, "multipleDiffsAgainstSingleTable");
+
+    Transaction transaction = catalog().buildTable(identifier, SCHEMA).create().newTransaction();
```

Comment:
I think it is easier to read if you separate actions on different lines. Here, I would expect a line where the table is created and a line that starts a transaction.